templates/firefox-ios.xliff

```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.Label.Title.v120" xml:space="preserve">
+        <source>How we determine review quality</source>
+        <note>Title of the 'How we determine review quality' card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.LearnMoreButton.Title.v120" xml:space="preserve">
+        <source>Learn more about how %@ determines review quality</source>
+        <note>The title of the learn more button from How we determine review quality card displayed in the shopping review quality bottom sheet. The placeholder will be replaced with the Fakespot app name.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.MixedReviews.Label.v120" xml:space="preserve">
+        <source>We believe there’s a mix of reliable and unreliable reviews</source>
+        <note>Mixed reviews label from How we determine review quality card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.ReliableReviews.Label.v120" xml:space="preserve">
+        <source>Reliable reviews. We believe the reviews are likely from real customers who left honest, unbiased reviews.</source>
+        <note>Reliable reviews label from How we determine review quality card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.SubHeadline.Label.v120" xml:space="preserve">
+        <source>We assign each product’s reviews a *letter grade* from A to F.</source>
+        <note>Label of the sub headline from How we determine review quality card displayed in the shopping review quality bottom sheet. The *text inside asterisks* denotes part of the string to bold, please leave the text inside the '*' so that it is bolded correctly.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.UnreliableReviews.Label.v120" xml:space="preserve">
+        <source>Unreliable reviews. We believe the reviews are likely fake or from biased reviewers.</source>
+        <note>Unreliable reviews label from How we determine review quality card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.Collapsed.AccessibilityLabel.v120" xml:space="preserve">
+        <source>Settings Card Collapsed</source>
+        <note>Accessibility label for the up chevron, from Settings Card View displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.Expanded.AccessibilityLabel.v120" xml:space="preserve">
+        <source>Settings Card Expanded</source>
+        <note>Accessibility label for the down chevron, from Settings Card View displayed in the shopping review quality bottom sheet.</note>
```

Comment:
same as above, I'm not sure this will be clear to localizers

Reply:
Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940